python/hello-world/README.adoc

```diff
@@ -0,0 +1,64 @@
+= School Timetabling (Python)
+
+Assign lessons to timeslots and rooms to produce a better schedule for teachers and students.
+
+== Prerequisites
+
+. Install Python 3.10+
+
+. Install JDK 17+, for example with https://sdkman.io[Sdkman]:
++
+----
+$ sdk install java
+----
+
+== Run the application
+
+. Git clone the timefold-quickstarts repo:
++
+[source, shell]
+----
+$ git clone https://github.com/TimefoldAI/timefold-quickstarts.git
+...
+$ cd timefold-quickstarts/python/hello-world
+----
+
+. Create a virtual environment
++
+[source, shell]
+----
+$ python -m venv .venv
+----
+
+. Activate the virtual environment
++
+[source, shell]
+----
+$ . .venv/bin/activate
+----
+
+. Install requirements
++
+[source, shell]
+----
+$ pip install -r requirements.txt
```

Comment:
I can't install the dependencies. Do I need to run any additional commands?
```
(.venv) fred@fedora:~/Projects/timefold/timefold-quickstarts/python/hello-world$ pip install -r requirements.txt
ERROR: Could not find a version that satisfies the requirement timefold-solver==999-dev0 (from versions: 1.10.0a0)
ERROR: No matching distribution found for timefold-solver==999-dev0
```

Reply:
You need to build https://github.com/TimefoldAI/timefold-solver-python , which will create the package in the `dist` folder. Then you need to tell `pip` to also look for packages in `/path/to/timefold-solver-python` using `--find-links`: `pip install --find-links /path/to/timefold-solver-python -r requirements.txt`.

For `stable`, this won't be needed (since it uses the PyPI package and not the SNAPSHOT package).

To avoid having to do `--find-links` everytime, you can add it to your `~/.config/pip/pip.conf`:

```
[global]
find-links =
    /path/to/timefold-solver-python/dist
```

Reply:
I have confirmed that it is working now. Thank you!